.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+kind/docs:
+  - docs/**/*
+
+kind/ui:
+  - web_src/**/*
+  - all: ["templates/**/*", "!templates/swagger/v1_json.tmpl"]
+
+kind/api:
+  - templates/swagger/v1_json.tmpl
+
+kind/build:
+  - Makefile
+  - Dockerfile
+  - Dockerfile.rootless
+  - webpack.config.js
+
+dependencies:
+  - go.mod
+  - go.sum
```

Comment:
If a PR imports a new module, this PR gets labeled as `dependencies`?

Reply:
I have removed the `dependencies` now for two reasons:

- The label does not follow the `kind/` prefix
- I don't want to go overboard with label spam